service/worker/scanner/data_corruption_workflow_test.go

```diff
@@ -43,6 +43,8 @@ import (
 
 const testWorkflowName = "default-test-workflow-type-name"
 
+var validBranchToken = []byte{89, 11, 0, 10, 0, 0, 0, 12, 116, 101, 115, 116, 45, 116, 114, 101, 101, 45, 105, 100, 11, 0, 20, 0, 0, 0, 14, 116, 101, 115, 116, 45, 98, 114, 97, 110, 99, 104, 45, 105, 100, 0}
```

Comment:
👍👍 for more valid values, though is there a more human-friendly way to build this?  as it apparently includes "test-tree-id" and "test-branch-id" strings

Reply:
Copied it from some other place, as the test actually started failing after checking for errors :)

This is thrift serialized structure underneath. I guess we could have helpers in our test to generate those.